impeller/display_list/display_list_dispatcher.cc

```diff
@@ -27,6 +27,59 @@ DisplayListDispatcher::DisplayListDispatcher() = default;
 
 DisplayListDispatcher::~DisplayListDispatcher() = default;
 
+static std::optional<Entity::BlendMode> ToBlendMode(flutter::DlBlendMode mode) {
+  switch (mode) {
```

Comment:
Instead of a large switch, is it possible to do a lookup in a map literal? Like:
```c++
const std::map<flutter::DlBlendMode, Entity::BlendMode> m = {
  {flutter::DlBlendMode::kClear, Entity::BlendMode::kClear},
  ...
};
return m[mode];
```

We do this sometimes in Dart because it can help the code be more readable since you can sort the keys and not worry about the quality of the generated code, but that might not be a concern in C++. You might also consider using a macro to make this easier to read:
```c++
#define SWITCH_CASE(a, b) case flutter::DlBlendMode::##a: return Entity::BlendMode::##b;
switch (mode) {
  SWITCH_CASE(kClear, kClear)
  SWITCH_CASE(kSrc, kSource)
  ...
}
#undef SWITCH_CASE
```

Maybe @chinmaygarde has an opinion =)

Reply:
Done.